test/cli/testdata/help/pub-downgrade.txt

```diff
@@ -0,0 +1,15 @@
+$ pub downgrade --help
+Downgrade the current package's dependencies to oldest versions.
+
+
+
+Usage: pub downgrade [dependencies...]
+-h, --help               Print this usage information.
+    --[no-]offline       Use cached packages instead of accessing the network.
```

Comment:
This just adds what we currently have... but I think we should do some follow-up PRs to improve these messages.
Like why do w need `--no-offline`? If we want to keep it for compatibility can we at-least hide the `--no-offline` variant.

I'm happy to keep `--offline`, and to keep it visible, it just seems the inverse is weird.

Reply:
I also don't think we need `--no-offline`.